module/VuFind/src/VuFind/Controller/InstallController.php

```diff
@@ -580,7 +580,8 @@ protected function checkILS()
             $status = false;
         } else {
             try {
-                $status = 'ils-offline' !== $this->getILS()->getOfflineMode(true);
+                $status = 'ils-offline' !== $this->getILS()->getOfflineMode(true)
+                    || ('NoILS' === $config->Catalog->driver);
```

Comment:
Maybe make this a little more error-proof?

```suggestion
                    || ('noils' === strtolower($config->Catalog->driver ?? ''));
```